Unity/opentok/holo/holo/ViewController.mm

```diff
@@ -665,9 +670,11 @@ - (void)subscriber:(nonnull OTSubscriberKit *)subscriber rtcStatsReport:(nonnull
     stats = [NSString stringWithFormat:@"Subscriber: %@\n", [inboundStats length] > 0 ? inboundStats : @"NA"];
     dispatch_async(dispatch_get_main_queue(), ^{
         [self->_subscriberStatsLabel removeFromSuperview];
-        [self->_subscriberStatsLabel setText:stats];
-        [self->_subscriberStatsLabel setNumberOfLines:0];
-        [[[[self unityFramework] appController] rootView]addSubview:self->_subscriberStatsLabel];
+        if(self->_enableLogs){
```

Comment:
Should also include `[self->_subscriberStatsLabel removeFromSuperview];` here?

Reply:
i dont think its a must but we will know once we test it.